src/app/me/contact/page.tsx

```diff
@@ -0,0 +1,11 @@
+import MyPageTitle from '@/app/me/_components/MyPageTitle';
+import ContactSection from '@/app/me/_components/ContactSection';
+
+export default function page() {
```

Comment:
```suggestion
export default function ContactPage() {
```
파스칼로 해야할거 같고, 어떤 페이지인지 명시적으로 적어주는것도 좋을거 같습니당~! 다른 페이지도!!